src/Components/GoodsList.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const GoodsList = ({ goods }) => (
+  <ul>
+    {goods.map(good => (
+      <li key={good}>
+        {good}
+      </li>
+    ))}
+  </ul>
+);
+
+GoodsList.propTypes = {
+  goods: PropTypes.arrayOf.isRequired,
```

Comment:
Явно вказуй, типи даних, які містить масив
```suggestion
  goods: PropTypes.arrayOf(PropType.string).isRequired,
```

Reply:
Погоджуюся з коментом Артура, додай опис масива